chapters/open_research.md

```diff
@@ -8,6 +8,61 @@
 
 Recommended skill level: low.
 
+
+## Table of contents
+
+1. [Summary](#Summary)
+2. [How this will help you/ why this is useful](#Why_this_is_useful)
+3. [Open data](#Open_data)
+    1. [Steps to make your data open](#Steps_to_make_your_data_open)
+        1. [Step 1: Make your data available](#Step_1_Make_your_data_available)
+        2. [Step 2: Make your data easy to understand](#Step_2_Make_your_data_easy_to_understand)
+        3. [Step 3: Make your data easy to use](#Step_3_Make_your_data_easy_to_use)
+        4. [Step 4: Make your data citeable](#citing_data)
+    2. [Barriers to data sharing](#Barriers_to_data_sharing)
+        1. [Privacy](#Privacy)
+        2. [National and commercially sensitive data](#National_and_commercially_sensitive_data)
+4. [Open source software](#Open_source_software)
+    1. [What is open source software](#What_is_open_source_software)
+    2. [How running and contributing to open source software projects benefits you](#How_running_and_contributing_to_open_source_software_projects_benefits_you)
+        1. [Making your own work open source](#Making_your_own_work_open_source)
+        2. [Contributing to others' open source software projects](#Contributing_to_others_open_source_software_projects)
+    3. [How open source software benefits research](#How_open_source_software_benefits_research)
+    4. [How to run your own open source software project](#How_to_run_your_own_open_source_software_project)
+        1. [Readme](#Readme)
+        2. [Contributing guidelines](#Contributing_guidelines)
+        3. [Code of conduct](#Code_of_conduct)
+    5. [How to contribute to other's open source software projects](How_to_contribute_to_others_open_source_software_projects)
```

Comment:
This link isn't working for me, but all others are fine

Reply:
Nice catch @LouiseABowler, it was broken due to a typo, should be fixed now